Doc/library/typing.rst

```diff
@@ -500,6 +504,8 @@ The module defines the following classes, functions and decorators:
 
     A generic version of :class:`collections.abc.Reversible`.
 
+   .. versionadded:: 3.5.2
```

Comment:
Since [`Reversible` was in 3.5.0](https://github.com/python/cpython/blob/v3.5.0/Lib/typing.py#L1317), the `versionadded` is unnecessary and `Misc/NEWS` should get fixed.